indico/modules/events/papers/views.py

```diff
@@ -16,22 +16,28 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.abstracts.views import _MathjaxMixin
```

Comment:
should be moved to a more suitable place than `abstracts`, and probably un-underscored